conf/cassandra.yaml

```diff
@@ -2622,13 +2622,23 @@ drop_compact_storage_enabled: false
 #
 storage_compatibility_mode: NONE
 
+# Prevents preparing a repair session or beginning a repair streaming session if pending compactions is over
+# the given value.  Defaults to disabled.
+# reject_repair_compaction_threshold: 1024
```

Comment:
Already existed, but there was no documentation so felt worth adding here since we also mention it in `auto_repair.adoc`

Reply:
![image](https://github.com/user-attachments/assets/69d9e911-85df-4466-b1cd-10626f344cfa)